src/app/screens/Login.tsx

```diff
@@ -1,30 +1,27 @@
-import React, { KeyboardEvent, useState } from 'react';
+import React, { FormEvent } from 'react';
+
+import { useForm } from 'typed-react-form';
 
 import { ROUTE_PATHS } from '@app/routing';
 import { login, useDispatch, useSelector } from '@app/store';
 import lock from '@assets/icons/lock.svg';
-import { Body, Box, Button, Flex, Heading, Input, Label, LinkApp, Logo } from '@components';
+import { Body, Box, Button, Flex, FormInput, Heading, Label, LinkApp, Logo } from '@components';
 import { Trans, translateRaw } from '@translations';
 
 export const Login = () => {
-  const [password, setPassword] = useState('');
+  const form = useForm({ password: '' });
   const error = useSelector((state) => state.auth.error);
   const dispatch = useDispatch();
 
-  const handleLogin = () => {
-    if (password.length > 0) {
-      dispatch(login(password));
-    }
-  };
+  const handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
 
-  const handleKeyPress = (event: KeyboardEvent<HTMLInputElement>) => {
-    if (event.key === 'Enter') {
-      handleLogin();
+    if (form.error) {
```

Comment:
How would there be an error without validation?

Reply:
This is a placeholder for when we add validation to the login page. I can add it as part of this PR.

Reply:
👍